config/locales/user_mailer/fr.yml

```
@@ -130,6 +130,16 @@ fr:
         identité en personne, vous devez apporter un justificatif de domicile
         valable si votre adresse actuelle est différente de celle figurant sur
         votre pièce d’identité.
+    in_person_failed_suspected_fraud:
+      body:
+        help_center_html: Si vous avez besoin d’aide supplémentaire, vous pouvez <a
+          href="%{help_center_url}">consulter notre centre d’aide</a> ou
+          contacter l’agence à laquelle vous essayez d’accéder.
+        intro: Nous comprenons que vous avez tenté de vérifier votre identité par le
+          biais de Login.gov, mais votre identité n’a pas pu être vérifiée au
```

```suggestion
          biais de %{app_name}, mais votre identité n’a pas pu être vérifiée au
```
We should always use `app_name` instead of hardcoding `Login.gov`